sessions/workshops/natural-language-processing-in-10-lines-of-code.md

```diff
@@ -0,0 +1,6 @@
+speaker: andraz-hribernik
+title: Natural Language Processing in 10 Lines of Code
+---
+Do you know that there are over 150 personal names mentioned in Jane Austen's Pride & Prejudice? Did you ever want to remove common words such as 'we', 'what', and 'for' from text? Or extract interesting keywords from it? Did you know all of this could be done in 10 lines of Python code?
+
+In this workshop we will answer these and similar questions using the open source library spaCy. One hour spent with us will be useful for anybody who is interested in text processing, text information extraction or text related data science. You do not need to have prior experiences with NLP or data science. However, some Python programming skills are needed. We would also encourage you to install spaCy in advance (link to Github repo with installation instructions).
```

Comment:
Looks like the speaker notes might be missing a link? Not sure if the speaker dropped it, or if it got lost somewhere in the process of getting to the site.